src/test/converter/RoundTrip.cpp

```diff
@@ -65,19 +67,20 @@ void RoundTrip::roundTripVersionedXmlFromMinToCurrentVersionTest(const std::stri
 }
 
 void RoundTrip::roundTripVersionedXmlTest(const std::string& filename, const iidm::converter::xml::IidmXmlVersion& version) {
-    const auto& writer = [&version](const iidm::Network& n, std::ostream& stream) {
+    const auto& writer = [&version, &filename](const iidm::Network& n, std::ostream& stream) {
         iidm::converter::ExportOptions options;
         options.setVersion(version.toString("."));
-        iidm::Network::writeXml(stream, n, options);
+        iidm::Network::writeXml(filename, stream, n, options);
     };
 
-    const auto& reader = [](const std::string& xml) {
-        std::istringstream stream(xml);
-        return iidm::Network::readXml(stream);
+    const auto& reader = [&filename](const std::string& networkStr) {
+        std::istringstream stream(networkStr);
+        return iidm::Network::readXml(filename, stream);
     };
```

Comment:
I think networkStr is a bad variable name: xml seems better to me if it really contains the XML content